### PR TITLE
added star ratings on hover for community feeds

### PR DESCRIPTION
**Pull request by gurjit03**
This PR fixes the issue #20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "babel-runtime": "6.26.0",
     "bluebird": "^3.5.1",
     "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "chalk": "1.1.3",
+    "chalk": "^1.1.3",
```
Comment (singhpratyush) [inline package.json]: This change is not required.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,13544 @@
+{
```
Comment (singhpratyush) [inline package-lock.json]: Please use `yarn` only.

File: src/post/ActionBar/index.js [hunk 3]
```diff
@@ -11,11 +11,14 @@ import { getAuthUsername } from '../../reducers/authUserReducer';
 import Icon from '../../icons/Icon';
```
Comment (singhpratyush) [inline src/post/ActionBar/index.js]: There are a lot of formatting changes in this file. If something is allowed by the `eslint` and `editorconfig`, I don't think we need to change it for no reason.

File: package.json (hunk 1 above)
Comment (gurjit03) [reply package.json]: I have taken care of it in current commit.

File: package-lock.json (hunk 2 above)
Comment (gurjit03) [reply package-lock.json]: sure, using only `yarn` now.

File: src/post/ActionBar/index.js (hunk 3 above)
Comment (gurjit03) [reply src/post/ActionBar/index.js]: I have take care of formatting changes as well in the recent commit, kindly have a look there.

**Comment by singhpratyush** [comment]
Awesome! Thanks for the contribution @gurjit03 :tada:

**Pull request merged**